plugins/notifications/ClickAppsVibrationsNotify.qml

```diff
@@ -36,7 +37,7 @@ ItemPage {
         anchors {
             left: parent.left
             right: parent.right
-            top: parent.top
+            top: parent.header.bottom
```

Comment:
Similar to the Sounds page, the margin at the top is too large:

![screenshot20181121_124252165](https://user-images.githubusercontent.com/21966173/48862311-97557f80-ed8b-11e8-972e-84fa75acc51b.png)

Reply:
Oh, sorry. that's because I added the flickable fix later in all files that required it without checking every page on the phone